README.md

```diff
@@ -2,12 +2,11 @@
 
 ## Running locally
 
-To run the application locally, install [pipenv](https://pipenv.readthedocs.io/en/latest/).  This app needs `python3` to run; if that's not your default python, adjust the command below (i.e. `python3` instead of `python`).
+To run the application locally, install [pipenv](https://pipenv.readthedocs.io/en/latest/).
 
 ```bash
 cd /path/to/this/repo
 pipenv install
-export FLASK_APP=application.py
```

Comment:
I still needed to set the `FLASK_APP` environment variable to get the app running, so I think we need to keep this line.

Reply:
Resolved in 68c9e7cdaef1bb3a7981f96f4cfc5db89328d7ee.